src/frontend/dune

```diff
@@ -22,26 +22,25 @@
 
 (rule
  (with-stdout-to parser_updated.messages
-  (run menhir %{dep:parser.mly} --update-errors %{dep:parser.messages}))
-)
+  (run menhir %{dep:parser.mly} --update-errors %{dep:parser.messages})))
 
-(alias
- (name update_messages)
-  (action (diff parser.messages parser_updated.messages)))
+(rule
+ (with-stdout-to parser_updated_trimmed.messages
+  (run python3 %{dep:strip_redundant_parser_state.py} %{dep:parser_updated.messages})))
```

Comment:
@seantalts what do you think about calling python3 explicitly here, instead of relying on the shebang? The issue being that my python installation isn't where the shebang points

Reply:
BTW, I used the script to update the messages for this branch and it worked great 👍